src/js/view/utils/submenu-factory.js

```diff
@@ -19,10 +19,15 @@ export const makeSubmenu = (settingsMenu, name, contentItems, icon) => {
     } else {
         const categoryButton = button(`jw-settings-${name}`, () => {
             settingsMenu.activateSubmenu(name);
+            console.log('submenu enter', submenu.element().children[0]);
```

Comment:
@hghazzi are we purposely leaving this console.log?

Reply:
Removed